catalog/NEON/NEON_CHM.jsonnet

```diff
@@ -0,0 +1,169 @@
+local id = 'NEON/CHM';
+local subdir = 'NEON';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+  ],
+  id: id,
+  title: 'NEON Canopy Height Model',
```

Comment:
This is still a conversation we're having, but would be good to discuss if/how it would possible to save multiple copies of data as updated data versions are created. We don't currently save multiple versions of AOP data due to funding limitations, but Kel mentioned that we may be able to save older versions in cold storage. Would like to follow up on this.

We are planning, for now, to include the data version (or what we call a "Data Release") in the RELEASE_TAG image property, so this would be saved at the image level, and so don't think it would make sense to include that in the jsonnet.